### ISSUE TITLE
incorrect output?

### ISSUE THREAD

**Issue by dmalan** [cs50/style50#11]
**Input**

```
#include <stdio.h>

int main(void)
    {
printf("hello, world\n");
    }
```

**Output**

<img width="399" alt="screen shot 2017-08-11 at 4 26 18 pm" src="https://user-images.githubusercontent.com/788678/29230647-e819dcd0-7eb1-11e7-9c26-6a753a4cc5ef.png">

**Comment by cmlsharp** [comment]
![](https://user-images.githubusercontent.com/3654575/29232072-bbcd1902-7eb7-11e7-97fe-5da6cbaf337f.png) 

I get this, which is correct, though somewhat unintuitive

**Comment by dmalan** [comment]
Hm, I still get what I screenshotted on both `master` and `develop`?

**Comment by cmlsharp** [comment]
Fixed in b0aae0e5a78823b3062e8b7d3f1d65cf7bd81202
